.github/workflows/examples.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: ["windows-latest", "ubuntu-latest", "macos-latest"]
+        os: ["windows-latest", "ubuntu-latest", "macos-13"]
```

Comment:
[fix] Remove `macos-latest` from required GitHub checks.

Reply:
and add macos-13 ones.